### ISSUE TITLE
bug Text/String/Number input ?

### ISSUE THREAD

**Issue by maxencelaurent** [Heigvd/Wegas#1448]
just watching the logs:
```

production-payara-1 | INFO : 07:04:52.779 [http-thread-pool::https-listener(1)] c.w.core.rest.util.ViewRequestFilter - Start Request [2861] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-1 | INFO : 07:04:52.782 [http-thread-pool::https-listener(1)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-1 | INFO : 07:04:52.817 [http-thread-pool::https-listener(1)] com.wegas.core.ejb.RequestManager - Request [2861] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 38 ms ( processing: 37; management: 0, propagation:  N/A, serialisation: 1) => No Content
production-payara-3 | INFO : 07:04:52.871 [http-thread-pool::https-listener(3)] c.w.core.rest.util.ViewRequestFilter - Start Request [3014] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-3 | INFO : 07:04:52.879 [http-thread-pool::https-listener(3)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-3 | INFO : 07:04:52.906 [http-thread-pool::https-listener(3)] com.wegas.core.ejb.RequestManager - Request [3014] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 36 ms ( processing: 35; management: 0, propagation:  N/A, serialisation: 1) => No Content
production-payara-2 | INFO : 07:04:52.935 [http-thread-pool::https-listener(8)] c.w.core.rest.util.ViewRequestFilter - Start Request [2940] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-2 | INFO : 07:04:52.938 [http-thread-pool::https-listener(8)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-2 | INFO : 07:04:52.957 [http-thread-pool::https-listener(8)] com.wegas.core.ejb.RequestManager - Request [2940] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 22 ms ( processing: 21; management: 0, propagation:  N/A, serialisation: 1) => No Content
production-payara-1 | INFO : 07:04:53.015 [http-thread-pool::https-listener(6)] c.w.core.rest.util.ViewRequestFilter - Start Request [2862] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-1 | INFO : 07:04:53.021 [http-thread-pool::https-listener(6)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-1 | INFO : 07:04:53.055 [http-thread-pool::https-listener(6)] com.wegas.core.ejb.RequestManager - Request [2862] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 40 ms ( processing: 39; management: 0, propagation:  N/A, serialisation: 1) => No Content
production-payara-3 | INFO : 07:04:53.197 [http-thread-pool::https-listener(2)] c.w.core.rest.util.ViewRequestFilter - Start Request [3015] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-3 | INFO : 07:04:53.201 [http-thread-pool::https-listener(2)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-3 | INFO : 07:04:53.233 [http-thread-pool::https-listener(2)] com.wegas.core.ejb.RequestManager - Request [3015] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 36 ms ( processing: 35; management: 0, propagation:  N/A, serialisation: 1) => No Content
production-payara-2 | INFO : 07:04:53.269 [http-thread-pool::https-listener(2)] c.w.core.rest.util.ViewRequestFilter - Start Request [2941] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-2 | INFO : 07:04:53.272 [http-thread-pool::https-listener(2)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-2 | INFO : 07:04:53.290 [http-thread-pool::https-listener(2)] com.wegas.core.ejb.RequestManager - Request [2941] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 21 ms ( processing: 21; management: 0, propagation:  N/A, serialisation: 0) => No Content
production-payara-1 | INFO : 07:04:53.338 [http-thread-pool::https-listener(7)] c.w.core.rest.util.ViewRequestFilter - Start Request [2863] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-1 | INFO : 07:04:53.339 [http-thread-pool::https-listener(7)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-1 | INFO : 07:04:53.381 [http-thread-pool::https-listener(7)] com.wegas.core.ejb.RequestManager - Request [2863] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 43 ms ( processing: 42; management: 0, propagation:  N/A, serialisation: 1) => No Content
production-payara-3 | INFO : 07:04:53.428 [http-thread-pool::https-listener(5)] c.w.core.rest.util.ViewRequestFilter - Start Request [3016] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-3 | INFO : 07:04:53.429 [http-thread-pool::https-listener(5)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-3 | INFO : 07:04:53.468 [http-thread-pool::https-listener(5)] com.wegas.core.ejb.RequestManager - Request [3016] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 40 ms ( processing: 40; management: 0, propagation:  N/A, serialisation: 0) => No Content
production-payara-2 | INFO : 07:04:53.498 [http-thread-pool::https-listener(1)] c.w.core.rest.util.ViewRequestFilter - Start Request [2942] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-2 | INFO : 07:04:53.500 [http-thread-pool::https-listener(1)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-2 | INFO : 07:04:53.516 [http-thread-pool::https-listener(1)] com.wegas.core.ejb.RequestManager - Request [2942] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 18 ms ( processing: 17; management: 0, propagation:  N/A, serialisation: 1) => No Content
production-payara-1 | INFO : 07:04:53.569 [http-thread-pool::https-listener(1)] c.w.core.rest.util.ViewRequestFilter - Start Request [2864] POST GameModel/35154/VariableDescriptor/Script/Run/14181962
production-payara-1 | INFO : 07:04:53.572 [http-thread-pool::https-listener(1)] com.wegas.core.rest.ScriptController - script for player 14181962 : Variable.find(gameModel, "initialBudget").setValue(self, 600000);
production-payara-1 | INFO : 07:04:53.607 [http-thread-pool::https-listener(1)] com.wegas.core.ejb.RequestManager - Request [2864] "POST GameModel/35154/VariableDescriptor/Script/Run/14181962" for [9322314::14181962::14183655] processed in 38 ms ( processing: 38; management: 0, propagation:  N/A, serialisation: 0) => No Content
production-payara-3 | INFO : 07:04:53.647 [http-thread-pool::https-listener(3)] 
```

**Comment by git-jarle** [comment]
StringInputs cannot be edited. Console messages:

react-dom.production.min.js:198 Error: Unhandled 'ReturnStatement'
    at t.visitNode (condition.js:62)
    at t.d.invokeVisitorMethod (path-visitor.js:284)
    at e.p.visitWithoutReset (path-visitor.js:159)
    at e.s.each (path.js:89)
    at f (path-visitor.js:180)
    at e.p.visitWithoutReset (path-visitor.js:168)
    at e.p.visit (path-visitor.js:106)
    at l.visit (path-visitor.js:82)
    at condition.js:47
    at Xo (react-dom.production.min.js:150)

**Comment by maxencelaurent** [comment]
fixed by 69868a679f3d4888d024d1e23025128e3c632f6c
